### PR TITLE
Fix kube-slack and monitoring hr

### PR DESCRIPTION
**Pull request by zc-hmcts**
- Add traefik azure identity

### CODE REVIEW

File: apps/admin/secrets-csi-driver/secrets-csi-driver-gitrepo.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: source.toolkit.fluxcd.io/v1beta1
-kind: HelmRepository
+kind: GitRepository
```
Comment (timja) [inline apps/admin/secrets-csi-driver/secrets-csi-driver-gitrepo.yaml:2]: this is actually a helm repository: https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/charts

Comment (zc-hmcts) [reply apps/admin/secrets-csi-driver/secrets-csi-driver-gitrepo.yaml]: think i just needed to update the url https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/charts/secrets-store-csi-driver/README.md

File: apps/admin/kube-slack/kube-slack.yaml [hunk 2]
```diff
@@ -7,6 +7,9 @@ metadata:
 spec:
   releaseName: kube-slack
   values:
+    valuesFrom:
```
Comment (timja) [inline apps/admin/kube-slack/kube-slack.yaml:10]: Isn’t valuesFrom part of the spec? Values is passed through to the chart not flux to retrieve from the sops secret

Comment (zc-hmcts) [reply apps/admin/kube-slack/kube-slack.yaml:10]: kured chart is using spec.values.valuesFrom and is deploying okay? https://github.com/hmcts/sds-flux-config/blob/master/apps/admin/kured/kured.yaml

**Review by timja — APPROVED** [review]

**Pull request merged**